src/registrar/templates/domain_request_additional_details.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'domain_request_form.html' %}
+{% load static field_helpers %}
+
+{% block form_instructions %}
+  <p><i>These questions are required (*).</i></p>
+{% endblock %}
+
+{% block form_required_fields_help_text %}
+{# commented out so it does not appear at this point on this page #}
+{% endblock %}
+
+<!-- TODO-NL: (refactor) Breakup into two separate components-->
+{% block form_fields %}
+    <fieldset class="usa-fieldset margin-top-2">
+      <legend>
+        <h2>Are you working with anyone from CISA regions on your domain request?</h2>
+        <p>.gov is managed by the Cybersecurity and Infrastructure Security Agency. CISA has 10 regions that some organizations choose to work with. Regional representatives use titles like protective security advisors, cyber security advisors, or election security advisors.</p>
```

Comment:
This is my bad as I only put it in the Figma and not in the content doc. But can we link "10 regions" like,

[10 regions](https://www.cisa.gov/about/regions)

Reply:
> This is my bad as I only put it in the Figma and not in the content doc. But can we link "10 regions" like,
> 
> [10 regions](https://www.cisa.gov/about/regions)

@zandercymatics Can we change this to a external link so it opens in a new tab and use the external link icon?